tests/net/lib/coap/src/main.c

```diff
@@ -990,6 +1058,7 @@ static int test_observer_client(void)
 		goto done;
 	}
 
+	coap_reply_clear(reply);
```

Comment:
Is there any issue without this line?

Reply:
This is done to do code coverage.

Reply:
No but did for code coverage.

Reply:
If that line is not related to particular test, please do not add just for code coverage.